.github/actions/docker-run/action.yml

```diff
@@ -59,11 +65,13 @@ runs:
         options: |
           -u ${{ env.RUNNER_UID }}:${{ env.RUNNER_GID }}
           -v ${{ github.workspace }}:/app/mtl
-          -v /home/ubuntu/.cache/huggingface:/home/ubuntu/.cache/huggingface
-          -v /home/ubuntu/.cache/matplotlib:/home/ubuntu/.cache/matplotlib
-          -v /home/ubuntu/.cache/pre-commit:/home/ubuntu/.cache/pre-commit
-          -v /home/ubuntu/.cache/torch:/home/ubuntu/.cache/torch
           --net=host
           ${{ inputs.docker_opts }}
+          -e LOGURU_LEVEL=${{ env.LOGURU_LEVEL }}
+          -e PYTHONPATH=/app/mtl
+          -e ARCH_NAME=${{ inputs.docker_image_arch }}
+          ${{ inputs.device }}
         run: |
+          cd /app/mtl
+          rm -rf tt_metal tt_eager
```

Comment:
Why are we deleting these?

Reply:
We do not want to mount the cache folders inside of the Docker container and have it be writeable by the user in a different OS. At least that is my opinion, too many issues with permissions. 

I am pretty open about this but feels wrong to me to map the host user's internal folder to the local Docker user.

Reply:
What is $ROOT in this case? Wouldn't this clear out a user's directory if it was mapped into the docker?

Reply:
If we're using this action with the dev / kitchen sink docker, I'd think this would get mapped into the docker from the outside?

Reply:
> Why are we deleting these?

Because they are conflicting with installed Python packages. Let's say we are importing `tt_metal` is that done from site-packages or from the current folder?

Reply:
Sounds good, should remove `ttnn` and as well then.